securityhub-org/sumologic-securityhub-collector-awsorg/sam/requirements.txt

```diff
@@ -1,2 +1,2 @@
-boto3==1.9.60
-requests==2.7.0
+boto3==1.27.1
```

Comment:
Can this be upgraded to latest version - 1.28.74

Reply:
I followed AWS's [recommendations](https://docs.aws.amazon.com/lambda/latest/dg/lambda-python.html)